docs/howto/writing-migrations.txt

```diff
@@ -86,11 +87,39 @@ the respective field according to your needs.
 * Add the field on your model with ``default=...`` and ``unique=True``
```

Comment:
I think we should simplify that paragraph to "with `default=uuid.uuid4` and `unique=True` arguments." and chop "In the example, ...."